lib/CustomFieldsFilters/utils.js

```diff
@@ -0,0 +1,33 @@
+import { buildArrayFieldQuery, buildDateRangeQuery } from '../AcqList';
+import {
+  CUSTOM_FIELDS_FILTER,
+  CUSTOM_FIELDS_TYPES,
+} from '../constants/customFields';
```

Comment:
` from "../constants"` should work as well since you export all from `customFields.js` (`export * from './customFields';`) in `../constants/index.js`, right?